Scheduler/observing_blocks_auxtel/latiss_daily_calibrations.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "latiss_daily_calibrations",
+    "program": "BIAS_DARK_FLAT",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "auxtel/enable_atcs.py",
```

Comment:
I think we should leave the `enable_atcs.py` and `enable_latiss.py` scripts out of this BLOCK. As part of the daily procedures, these should be enabled independently (in case of failure).